### DISCUSSION TITLE
Text field with space converts to date when using save row in MySQL database

### DISCUSSION THREAD

**Discussion by ybotmallah** [Budibase/budibase#5639] [Help] [answered]
Hi,

Fairly sure that this is a bug, but wondering if someone could verify / recreate this.

When entering a value that contains a space and using the save row action (and having column values set), the text values covert to date strings? I am using the MySQL database connector, I've included some screenshots below.

<img width="567" alt="image" src="https://user-images.githubusercontent.com/19497751/165804970-fd00a617-1b8b-46b3-9438-6c9f3d3cf7e9.png">
... becomes ...
<img width="918" alt="image" src="https://user-images.githubusercontent.com/19497751/165805425-2d00eb23-c4e7-4d76-9617-f6b44725dd02.png">
You can see that the text fields with no space have entered in fine.

<img width="506" alt="image" src="https://user-images.githubusercontent.com/19497751/165805150-f95493c7-4f40-4052-be10-ebfc88dc34aa.png">
Save row action contains column mappings, I am unsure if this makes a difference. In another application I've made I also faced this issue, however adding another column mapping and wrapping the value in titleize handlebars resolved the issue.

![2022-04-28_18-03-14](https://user-images.githubusercontent.com/19497751/165806683-a60d388e-804e-48c2-8cfd-cc7c64f0b1aa.png)
The table setup in MySQL

**Comment by ybotmallah** [comment]
A few more screenshots that might be useful on figuring out what's happening, I took these from the developer tools network tab.

<img width="958" alt="2022-04-29_14-04-40" src="https://user-images.githubusercontent.com/19497751/165972133-7f444dc0-dd73-4289-bb3c-b3f190be350c.png"> You can see that here the text is correct in the payload, however clicking on the preview tab over it is shown as a date string? Is this a bug in the code that parses the insert row action?

<img width="956" alt="2022-04-29_14-04-52" src="https://user-images.githubusercontent.com/19497751/165972297-926e5d46-984c-43c2-b32e-e226ab56eac7.png"> You can clearly see that what was text in the previous tab has converted to a date string here.

**Comment by melohagan** [comment, marked as answer]
Hi @ybotmallah 

I think this issue may be related: https://github.com/Budibase/budibase/issues/5612

**Reply by ybotmallah** [reply]
Hi again @melohagan, you look to be correct! I've marked your reply as the answer. Thank you.
